app/src/pages/association.tsx

```diff
@@ -89,7 +89,7 @@ export const getServerSideProps: GetServerSideProps<{
   return {
     props: {
       association: await directus().request(
-        readSingleton("association", queryTranslations)
+        readSingleton("association", queryTranslations as any)
```

Comment:
Why do we need this now? Removing a warning?

Reply:
There is a type error because the signature of the `readSingleton` function is incorrect. There is nothing we can do about it, except casting to `any` :/